### ISSUE TITLE
custom audience add_users is sending hashed advertiser ids

### ISSUE THREAD

**Issue by Sakedo** [facebook/facebook-python-ads-sdk#259]
The default behavior for <audience id>.add_users is to hash the payload. 

The default values for audiences based on MOBILE_ADVERTISER_ID should be to lower case the strings, leave in the dashes, and send the payload in the clear.

You can specify the pre_hashed parameter to work around the issue like this:
audience.add_users(CustomAudience.Schema.mobile_advertiser_id, users, pre_hashed=True)

**Comment by emetselaar** [comment]
Yes - I can verify that this is indeed the case. Took me quite some time to figure out what was going on.

Should the library perhaps be changed so that the hashing does not happen when choosing mobile_advertiser_id ?
